resources/views/pages-legacy/reorderSiteAwards.blade.php

```diff
@@ -108,6 +82,9 @@ function postAllAwardsDisplayOrder(awards) {
 
             <div class="flex w-full items-center justify-between mt-3 mb-6">
                 <div class="flex items-center gap-x-1">
+                    <div class="ml-auto mr-4">
```

Comment:
CSS: I think we should remove this div and handle element positioning with flexbox. It seems like what you're trying to create is a 1rem gap between the button and the checkbox. This is more maintainable when lifted up into a `gap` class, eg:

```html
<div class="flex items-center gap-4">
    <button onclick="...">Reset Order</button>
    <label class="flex...">
        <input ... />
    </label>
</div>
```